### DISCUSSION TITLE
v0.12.0-beta.3

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#745] [Releases]
## What's Changed
* [Bugfix] Missing "v" from build versions by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/743
* Release v0.12.0-beta.3 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/744

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.12.0-beta.2...v0.12.0-beta.3

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.12.0-beta.3'>v0.12.0-beta.3</a>.</em>

**Comment by repcsi** [comment]
Looks to be there :) 

Digest: sha256:a6104be1544b4d44c5f994bf527dae012a5a45b6affcaaade3749fb440c98525
Status: Downloaded newer image for ghcr.io/alexjustesen/speedtest-tracker:v0.12.0-beta.3
ghcr.io/alexjustesen/speedtest-tracker:v0.12.0-beta.3
$ docker image ls
REPOSITORY                                               TAG                   IMAGE ID       CREATED          SIZE
ghcr.io/alexjustesen/speedtest-tracker                   v0.12.0-beta.3        0b48703cf9fe   43 minutes ago   324MB
ghcr.io/alexjustesen/speedtest-tracker                   latest                edc453807258   2 days ago       324MB

**Reply by alexjustesen** [reply]
niiice

**Comment by ShlomiD83** [comment]
Wouldn't it be better to have a general Beta tag instead of beta+version?

**Reply by alexjustesen** [reply]
Depends on the pipeline, in this project's case I prefer the `-beta.x` prefix for `prereleases` so that I can test builds in the pipeline.

**Comment by jiriteach** [comment]
Working well - APP_KEY issue on start-up. Restart sorted that. I saw your PR for that one - should resolve once its in the next build. Great work!
